### PR TITLE
Facade

### PR DESCRIPTION
**Pull request by YuliaKolp**
Реализация фасадов.
Не уверена как красиво сделать реализацию со сменой email и тестом на неверный токен - разве что сделать еще два хелпера, но слогикой ошибок

### CODE REVIEW

File: helpers/account_helper.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            'login': login,
+            'password': password,
+            'rememberMe': remember_me,
+        }
+        response = self.dm_api_account_api.login_api.post_v1_account_login(json_data=json_data)
+        assert response.status_code == 200, f"User cannot authorise {response.json()}"
+        return response
+
+    # Костыль - Не знаю, как красиво переопределить (?) метод user_login
+    def user_login_wrong_password(
```
Comment (ValeriyMenshikov) [inline helpers/account_helper.py:55]: Не надо переопределять, у нас нет цели все запрятать в хэлпер, вызови обычный user_login и через
pytest.raises или через try: except: провалидируй статус, код. Этого тут точно не должно быть.

File: tests/functional/post_v1_account/test_post_v1_account_login.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    password = '123456789'
+    email = f'{login}@mail.ru'
+
+    account_helper.register_new_user(login=login, password=password, email=email)
+
+    # TODO add parametrized tests with empty password and password belong to another user
+
+    # авторизоваться c неверным паролем
+
+    account_helper.user_login_wrong_password(login=login, password=password)
```
Comment (ValeriyMenshikov) [inline tests/functional/post_v1_account/test_post_v1_account_login.py:39]: а почему бы с верным паролем не авторизоваться?

File: tests/functional/post_v1_account/test_put_v1_account.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
+from json import loads
+
+from dm_api_account.apis.account_api import AccountApi
+from dm_api_account.apis.login_api import LoginApi
+from api_mailhog.apis.mailhog_api import MailhogApi
+from utils import utils
+from restclient.configuration import Configuration as MailhogConfiguration
+from restclient.configuration import Configuration as DmApiConfiguration
+
+def test_put_v1_account():
```
Comment (ValeriyMenshikov) [inline tests/functional/post_v1_account/test_put_v1_account.py:10]: очень странное хранение тестов папка post_v1_account, модуль test_put_v1_account , при этом проверяем по факту test_put_v1_account_email вероятно. 
Надо привести все папки в консистентное состояние и тест перевести на использоваение хэлпера, слишком много низкоуровневого кода в тесте.

File: tests/functional/post_v1_account/test_put_v1_account_token.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+from dm_api_account.apis.account_api import AccountApi
+from dm_api_account.apis.login_api import LoginApi
+from api_mailhog.apis.mailhog_api import MailhogApi
+from utils import utils
+from restclient.configuration import Configuration as MailhogConfiguration
+from restclient.configuration import Configuration as DmApiConfiguration
+
+
+def test_put_v1_account_token():
```
Comment (ValeriyMenshikov) [inline tests/functional/post_v1_account/test_put_v1_account_token.py:11]: тоже переписать с использование хэлпера

File: tests/functional/post_v1_account/test_post_v1_account_login.py (hunk 2 above)
Comment (YuliaKolp) [reply tests/functional/post_v1_account/test_post_v1_account_login.py:39]: Нужна же проверка , что тест с неверным условием упадет

File: helpers/account_helper.py (hunk 1 above)
Comment (YuliaKolp) [reply helpers/account_helper.py:55]: В упор не пойму как переделать в хелпере  `assert response.status_code == 200`  
В test_post_v1_account_login.py делать вход с неверным паролем через pytest.raises

File: tests/functional/post_v1_account/test_post_v1_account_login.py [hunk 5]
```diff
@@ -36,5 +54,7 @@ def test_post_v1_account_login():
 
     # авторизоваться c неверным паролем
 
-    account_helper.user_login_wrong_password(login=login, password=password)
-
+    try:
+        response = account_helper.user_login(login=login, password=f'{password}_WRONG')
+    except:
```
Comment (ValeriyMenshikov) [inline tests/functional/post_v1_account/test_post_v1_account_login.py:59]: можно просто вынести ассерт сюда

assert response.status_code == 403
